lib/puppet_litmus/serverspec.rb

```diff
@@ -51,10 +51,9 @@ def create_manifest_file(manifest)
       manifest_file_location = manifest_file.path
     else
       # transfer to TARGET_HOST
-      command = "bundle exec bolt file upload #{manifest_file.path} /tmp/#{File.basename(manifest_file)} --nodes #{target_node_name} --inventoryfile inventory.yaml"
-      stdout, stderr, status = Open3.capture3(command)
-      error_message = "Attempted to run\ncommand:'#{command}'\nstdout:#{stdout}\nstderr:#{stderr}"
-      raise error_message unless status.to_i.zero?
+      inventory_hash = inventory_hash_from_inventory_file
+      result = upload_file(manifest_file.path, "/tmp/#{File.basename(manifest_file)}", target_node_name, options: {}, config: nil, inventory: inventory_hash)
+      raise result.first['result'].to_s unless result.first['status'] == 'success'
 
       manifest_file_location = "/tmp/#{File.basename(manifest_file)}"
```

Comment:
Probably nicer to define this variable beforehand and then use it as an argument in `upload_file`

Reply:
are you talking about `manifest_file_location` ?

Reply:
Yes. So define it before or after `inventory_hash`.